third-party/realdds/include/realdds/dds-device-impl.h

```diff
@@ -51,6 +53,51 @@ std::ostream & operator<<( std::ostream & s, state_type st )
 
 }  // namespace
 
+class dds_device::dds_stream::impl
+{
+public:
+    impl( rs2_stream type, std::string group_name )
+        : _type( type )
+        , _group_name( group_name )
+    {
+    }
+
+    rs2_stream _type;
+    std::string _group_name;
+
+    std::vector< std::pair< rs2_video_stream, bool > > _video_profiles;
```

Comment:
Why do we need both video and motion profiles per stream? Isn't it one or the other?
Should we not resolve this at stream-creation time? I.e., have a `video_stream` and a `motion_stream`?

Reply:
The struct is different, motion does not have width/highest , at least that's why it was separated in the IDL

Reply:
Created `dds_video_stream` and `dds_motion_stream`